jasmine/BUILD.bazel

```diff
@@ -28,4 +28,8 @@ bzl_library(
     name = "defs",
     srcs = ["defs.bzl"],
     visibility = ["//visibility:public"],
+    deps = [
+        "@aspect_rules_jasmine//jasmine/private:jasmine_test",
```

Comment:
shouldn't need `@aspect_rules_jasmine` within the local repo

Reply:
somehow stardoc gets confused and errors with 

```
Stardoc documentation generation failed: File jasmine/defs.bzl imported '@aspect_rules_jasmine//jasmine/private:jasmine_test.bzl', yet jasmine/private/jasmine_test.bzl was not found, even at roots [., external/aspect_rules_jasmine].
```

i am not sure what causes this.

Reply:
Oh nvm. found the reason.